### ISSUE TITLE
Flaky test:  Edit contacts with the default config should remove the primary contact from the clinic when the contact is deleted

### ISSUE THREAD

**Issue by dianabarsan** [medic/cht-core#8097]
**Describe the issue**
Test is failing frequently and repeatedly on retries: 
```
------------------------------------------------------------------
[chrome 110.0.5481.100 linux #0-7] Running: chrome (v110.0.5481.100) on linux
[chrome 110.0.5481.100 linux #0-7] Session ID: 339f9ca4-2d38-4792-8357-0dfc4b70efe3
[chrome 110.0.5481.100 linux #0-7]
[chrome 110.0.5481.100 linux #0-7] » /tests/e2e/default/contacts/edit.wdio-spec.js
[chrome 110.0.5481.100 linux #0-7] Edit contacts with the default config. 
[chrome 110.0.5481.100 linux #0-7]    ✓ should edit contact name
[chrome 110.0.5481.100 linux #0-7]    ✖ should remove the primary contact from the clinic when the contact is deleted
[chrome 110.0.5481.100 linux #0-7]    ✖ should remove the primary contact from the clinic when the contact is deleted
[chrome 110.0.5481.100 linux #0-7]    ✖ should remove the primary contact from the clinic when the contact is deleted
[chrome 110.0.5481.100 linux #0-7]    ✖ should remove the primary contact from the clinic when the contact is deleted
[chrome 110.0.5481.100 linux #0-7]    ✖ should remove the primary contact from the clinic when the contact is deleted
[chrome 110.0.5481.100 linux #0-7]    ✖ should remove the primary contact from the clinic when the contact is deleted
[chrome 110.0.5481.100 linux #0-7]
[chrome 110.0.5481.100 linux #0-7] 1 passing (58.1s)
[chrome 110.0.5481.100 linux #0-7] 1 failing
[chrome 110.0.5481.100 linux #0-7]
[chrome 110.0.5481.100 linux #0-7] 1) Edit contacts with the default config.  should remove the primary contact from the clinic when the contact is deleted
[chrome 110.0.5481.100 linux #0-7] element ("#contact_summary") still not displayed after 15000ms
[chrome 110.0.5481.100 linux #0-7] Error: element ("#contact_summary") still not displayed after 15000ms
[chrome 110.0.5481.100 linux #0-7]     at async Object.waitForContactLoaded (/home/runner/work/cht-core/cht-core/tests/page-objects/default/contacts/contacts.wdio.page.js:126:3)
[chrome 110.0.5481.100 linux #0-7]     at async Context.<anonymous> (/home/runner/work/cht-core/cht-core/tests/e2e/default/contacts/edit.wdio-spec.js:55:5)
[chrome 110.0.5481.100 linux #0-7]
[chrome 110.0.5481.100 linux #0-7] 2) Edit contacts with the default config.  should remove the primary contact from the clinic when the contact is deleted
[chrome 110.0.5481.100 linux #0-7] Waiting for Loading spinners to hide timed out.
[chrome 110.0.5481.100 linux #0-7] Error: Waiting for Loading spinners to hide timed out.
[chrome 110.0.5481.100 linux #0-7]     at async waitForLoaders (/home/runner/work/cht-core/cht-core/tests/page-objects/default/common/common.wdio.page.js:170:3)
[chrome 110.0.5481.100 linux #0-7]     at async waitForPageLoaded (/home/runner/work/cht-core/cht-core/tests/page-objects/default/common/common.wdio.page.js:187:5)
[chrome 110.0.5481.100 linux #0-7]     at async Object.goToPeople (/home/runner/work/cht-core/cht-core/tests/page-objects/default/common/common.wdio.page.js:103:5)
[chrome 110.0.5481.100 linux #0-7]     at async Context.<anonymous> (/home/runner/work/cht-core/cht-core/tests/e2e/default/contacts/edit.wdio-spec.js:43:5)
[chrome 110.0.5481.100 linux #0-7]
[chrome 110.0.5481.100 linux #0-7] 3) Edit contacts with the default config.  should remove the primary contact from the clinic when the contact is deleted
[chrome 110.0.5481.100 linux #0-7] expected [] to have the same members as [ 'Ana Paula Gonzalez' ]      
      actual expected
      
      [ 'Ana Paula Gonzalez' ]
      
[chrome 110.0.5481.100 linux #0-7] AssertionError: expected [] to have the same members as [ 'Ana Paula Gonzalez' ]
[chrome 110.0.5481.100 linux #0-7]     at Context.<anonymous> (/home/runner/work/cht-core/cht-core/tests/e2e/default/contacts/edit.wdio-spec.js:48:62)
[chrome 110.0.5481.100 linux #0-7]     at processTicksAndRejections (node:internal/process/task_queues:96:5)
[chrome 110.0.5481.100 linux #0-7]
[chrome 110.0.5481.100 linux #0-7] 4) Edit contacts with the default config.  should remove the primary contact from the clinic when the contact is deleted
[chrome 110.0.5481.100 linux #0-7] expected [] to have the same members as [ 'Ana Paula Gonzalez' ]      
      actual expected
      
      [ 'Ana Paula Gonzalez' ]
      
[chrome 110.0.5481.100 linux #0-7] AssertionError: expected [] to have the same members as [ 'Ana Paula Gonzalez' ]
[chrome 110.0.5481.100 linux #0-7]     at Context.<anonymous> (/home/runner/work/cht-core/cht-core/tests/e2e/default/contacts/edit.wdio-spec.js:48:62)
[chrome 110.0.5481.100 linux #0-7]     at runMicrotasks (<anonymous>)
[chrome 110.0.5481.100 linux #0-7]     at processTicksAndRejections (node:internal/process/task_queues:96:5)
[chrome 110.0.5481.100 linux #0-7]
[chrome 110.0.5481.100 linux #0-7] 5) Edit contacts with the default config.  should remove the primary contact from the clinic when the contact is deleted
[chrome 110.0.5481.100 linux #0-7] expected [] to have the same members as [ 'Ana Paula Gonzalez' ]      
      actual expected
      
      [ 'Ana Paula Gonzalez' ]
      
[chrome 110.0.5481.100 linux #0-7] AssertionError: expected [] to have the same members as [ 'Ana Paula Gonzalez' ]
[chrome 110.0.5481.100 linux #0-7]     at Context.<anonymous> (/home/runner/work/cht-core/cht-core/tests/e2e/default/contacts/edit.wdio-spec.js:48:62)
[chrome 110.0.5481.100 linux #0-7]     at runMicrotasks (<anonymous>)
[chrome 110.0.5481.100 linux #0-7]     at processTicksAndRejections (node:internal/process/task_queues:96:5)
[chrome 110.0.5481.100 linux #0-7]
[chrome 110.0.5481.100 linux #0-7] 6) Edit contacts with the default config.  should remove the primary contact from the clinic when the contact is deleted
[chrome 110.0.5481.100 linux #0-7] expected [] to have the same members as [ 'Ana Paula Gonzalez' ]      
      actual expected
      
      [ 'Ana Paula Gonzalez' ]
      
[chrome 110.0.5481.100 linux #0-7] AssertionError: expected [] to have the same members as [ 'Ana Paula Gonzalez' ]
[chrome 110.0.5481.100 linux #0-7]     at Context.<anonymous> (/home/runner/work/cht-core/cht-core/tests/e2e/default/contacts/edit.wdio-spec.js:48:62)
[chrome 110.0.5481.100 linux #0-7]     at runMicrotasks (<anonymous>)
[chrome 110.0.5481.100 linux #0-7]     at processTicksAndRejections (node:internal/process/task_queues:96:5)
------------------------------------------------------------------
```

Example run: https://github.com/medic/cht-core/actions/runs/4246635487/jobs/7397413956#step:16:576

**Comment by garethbowen** [comment]
Success rate as of today 100%

**Comment by dianabarsan** [comment]
I'm getting lots of failures for this.

**Comment by tatilepizs** [comment]
Fixed on PR #9221
